src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/PersonDetailsControllerTest.kt

```diff
@@ -16,7 +16,7 @@ class PersonDetailsControllerTest(
 ) : IntegrationTestBase() {
 
   @Test
-  fun `retrieves person details`() {
+  fun `retrieves recommendation`() {
```

Comment:
Is this test name supposed to be changed?

Reply:
Corrected!